src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -0,0 +1,4017 @@
+/**
+ * (C) Copyright 2016-2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#include <stdio.h>
+#include <assert.h>
+#include <dirent.h>
+#include <dlfcn.h>
+#include <sys/types.h>
+#include <utime.h>
+#include <sys/stat.h>
+#include <sys/vfs.h>
+#include <fcntl.h>
+#include <stdarg.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/statfs.h>
+#include <sys/statvfs.h>
+#include <pthread.h>
+#include <sys/ioctl.h>
+
+#include <gurt/list.h>
+#include <gurt/common.h>
+#include <gurt/hash.h>
+#include <daos.h>
+#include <daos_fs.h>
+#include <dfuse_ioctl.h>
+#include <daos_prop.h>
+#include "dfs_internal.h"
+
+#include "hook.h"
+
+/* Use very large synthetic FD to distinguish regular FD from Kernel */
+#define FD_FILE_BASE		(0x20000000)
+#define FD_DIR_BASE		(0x40000000)
+#define DUMMY_FD_DIR		(0x50000000)
```

Comment:
what is this for? add a comment maybe?

Reply:
I removed DUMMY_FD_DIR and added comments for FD_FILE_BASE and FD_DIR_BASE.